### PR TITLE
Implement kubermatic-installer deploy seed-mla

### PR DESCRIPTION
**Pull request by mohamed-rafraf**
**What this PR does / why we need it**:

Install the Master/Seed MLA stack with kubermatic-installer

**Which issue(s) this PR fixes**:

Fixes #[5860](https://github.com/kubermatic/kubermatic/issues/5860) and #[5861](https://github.com/kubermatic/kubermatic/issues/5861) 

**What type of PR is this?**

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
The `kubermatic deploy seed-mla` subcommand installs the Master/Seed MLA stack. Use `--skip-charts` to exclude specific helm charts from the logging and monitoring stack during installation.
```

### CODE REVIEW

**Comment by mohamed-rafraf** [comment]
image

File: pkg/install/stack/seed-mla/validation.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	return nil
+}
+
+func isBlockSecret(value string) bool {
+	switch len(value) {
+	case 16, 24, 32:
+		return true
+	}
+	return false
+}
```
Comment (xrstf) [inline pkg/install/stack/seed-mla/validation.go:99]: Can you leave a comment or some sort of explanation where this is coming from? I think you might have copied it from the usercluster MLA (fair enough), but please tell us so.

File: pkg/install/stack/seed-mla/stack.go [hunk 2]
```diff
@@ -0,0 +1,489 @@
...
+	return "KKP Seed MLA Stack"
+}
+
+func (s *Monitoring) Deploy(ctx context.Context, opt stack.DeployOptions) error {
+	if err := deployNodeExporter(ctx, opt.Logger, opt.KubeClient, opt.HelmClient, opt); err != nil {
+		return fmt.Errorf("failed to deploy Node Exporter: %w", err)
+	}
+
+	if err := deployKubeStateMetrics(ctx, opt.Logger, opt.KubeClient, opt.HelmClient, opt); err != nil {
+		return fmt.Errorf("failed to deploy Kube State Metrics: %w", err)
```
Comment (xrstf) [inline pkg/install/stack/seed-mla/stack.go]: This is more commonly known as "kube-state-metrics"

File: pkg/install/stack/seed-mla/stack.go [hunk 3]
```diff
@@ -0,0 +1,489 @@
...
+	if err := deployGrafana(ctx, opt.Logger, opt.KubeClient, opt.HelmClient, opt); err != nil {
+		return fmt.Errorf("failed to deploy Grafana: %w", err)
+	}
+
+	if err := deployBlackboxExporter(ctx, opt.Logger, opt.KubeClient, opt.HelmClient, opt); err != nil {
+		return fmt.Errorf("failed to deploy Blackbox Exporter: %w", err)
+	}
+
+	if err := deployAlertManager(ctx, opt.Logger, opt.KubeClient, opt.HelmClient, opt); err != nil {
+		return fmt.Errorf("failed to deploy Alert Manager: %w", err)
```
Comment (xrstf) [inline pkg/install/stack/seed-mla/stack.go]: Alertmanager is a product name

**Review by xrstf — CHANGES_REQUESTED** [review]
Just some minor nitpicks :)

**Comment by xrstf** [comment]
Can you please add a release-note about the new command and what it does?

File: pkg/install/stack/seed-mla/validation.go (hunk 1 above)
Comment (mohamed-rafraf) [reply pkg/install/stack/seed-mla/validation.go:99]: Done

File: pkg/install/stack/seed-mla/stack.go (hunk 2 above)
Comment (mohamed-rafraf) [reply pkg/install/stack/seed-mla/stack.go]: Done

File: pkg/install/stack/seed-mla/stack.go (hunk 3 above)
Comment (mohamed-rafraf) [reply pkg/install/stack/seed-mla/stack.go]: Done

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**